server/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -233,15 +234,7 @@
     private final ValuesSourceRegistry valuesSourceRegistry;
     private final TimestampFieldMapperService timestampFieldMapperService;
     private final CheckedBiConsumer<ShardSearchRequest, StreamOutput, IOException> requestCacheKeyDifferentiator;
-
```

Comment:
I moved this method below the ctor and added the time series id generator service as a cluster state consumer. Having instance methods above the ctor confuses me.